src/xrServerEntities/script_ini_file.cpp

```diff
@@ -213,6 +217,22 @@ void CScriptIniFile::set_override_names(bool b)
     inherited::set_override_names(b);
 }
 
+void CScriptIniFile::section_for_each(const luabind::functor<void>& functor)
+{
+    using sections_type = CInifile::Root;
+    sections_type& sections = this->sections();
```

Comment:
Never use this, only if it's really necessary..
```suggestion
    sections_type& sections = sections();
```

Reply:
Also, is it possible to make it `const`?

Reply:
Fixed